README.md

```diff
@@ -134,9 +134,7 @@ apt-get install                           \
 This project strives to be as normal and simple a CMake project as possible. This build workflow in particular will work, producing a static `libbeman.exemplar.a` library, ready to package with its headers:
 
 ```shell
-cmake -B build -S . -DCMAKE_CXX_STANDARD=20
-cmake --build build
-ctest --test-dir build
+cmake --workflow --preset gcc-debug
 cmake --install build --prefix /opt/beman.exemplar
 ```
 
```

Comment:
Note that the "verbose logs" sections below need to be updated as well.

On the other hand, I'd be happier to see them removed entirely. I've used beman.exemplar for a few different projects now and it's a pain to have to update those sections.

The "Disable tests build" section also needs to be updated.

Reply:
Thank you for double checking! Sorry I missed those sections. I will have them updated later today. !!

Reply:
Should I simply remove the verbose logs then?

Reply:
Updated cmake command and output in 84305172eb1af8ef7b7b7d0b9b17f0f3a5aa7c01 .